packages/react-components/src/components/MessageStatusIndicator.tsx

```diff
@@ -96,6 +97,7 @@ export const MessageStatusIndicator = (props: MessageStatusIndicatorProps): JSX.
           data-ui-id="chat-composite-message-tooltip"
           calloutProps={{ ...calloutProps }}
         >
+          <LiveMessage message={strings.failedToSendAriaLabel ?? ''} aria-live="polite" />
```

Comment:
Is this just a announcer?

Reply:
Yeah basically, we use this in other places in the code for annoucing information

Reply:
Would like to note that **LiveMessage** needs to be wrapped in **LiveAnnouncer** in some shape or form.
So if using this in other places, make sure to double check that **LiveAnnouncer** is living as a parent, if not, make it a parent.

Pasting my other comment here for reference:


Just a note for future reference:
Noting that LiveMessage needs to be wrapped in a LiveAnnouncer for it to work, according to the docs: 
https://www.npmjs.com/package/react-aria-live

Followed the trail and found that MessageThread.tsx has LiveAnnouncer which wraps MessageStatusIndicator like so:
\<LiveAnnouncer\> 
**->** \<Chat\> 
**->** item=MessageToDisplay 
**->** itemOnRenderMessageStatus 
**->** memoizeAllMessages 
**->** {message: content: ... defaultStatusRenderer} 
**->** \<MessageStatusIndicator\> 
**->** \<LiveMessage\>

(As of June 2, 2022, things may have changed in between the time you read this comment)